typescript/packages/well-completions-plot/src/WellCompletionsPlot.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { PlotData, WellCompletionsPlot } from "./WellCompletionsPlot";
+
+import {
+    timeSteps,
+    firstPlotData,
+    secondPlotData,
+    thirdPlotData,
+} from "../example-data/well-completions-plot-data";
+
+/**
+ * Storybook test for the well completions plot component
+ */
+export default {
+    component: WellCompletionsPlot,
+    title: "WellCompletionsPlot/Demo",
+    argTypes: {
+        data: {
+            control: {
+                type: "object",
+            },
+        },
+    },
+};
+
+const Template = (data) => {
+    const [plotData, setPlotData] = React.useState<PlotData>(data.plotData);
+    const [timeSteps, setTimeSteps] = React.useState<string[]>(data.timeSteps);
+
+    React.useEffect(() => {
+        setPlotData(data.plotData);
+    }, [data.plotData]);
+    React.useEffect(() => {
+        setTimeSteps(data.timeSteps);
+    }, [data.timeSteps]);
```

Comment:
According to https://react.dev/learn/you-might-not-need-an-effect, consider removing the effects and store the previous states.